{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"pymongo.errors.OperationFailure: Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1726623408, 5), 'signature': {'hash': b'\\x97\\x1bm/C4\\xacC\\xec%gr\\xe9<{\\xb3\\x18\\x86\\r\\x91', 'keyId': 7415789480730165257}}, 'operationTime': Timestamp(1726623408, 5)}","children":[{"name":"test_mongos_external_connections","uid":"f56b2fc7c1f9508d","parentUid":"8111ead8fe1db579aea8904d0f4b1e11","status":"broken","time":{"start":1726623388617,"stop":1726623408805,"duration":20188},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":true,"parameters":[],"tags":["asyncio","abort_on_fail","group(1)"]}],"uid":"8111ead8fe1db579aea8904d0f4b1e11"}],"uid":"bdbf199525818fae7a8651db9eafe741"},{"name":"Product defects","children":[{"name":"AssertionError: Mongos is not currently running.\nassert False","children":[{"name":"test_mongos_has_user","uid":"c93f4ba5b8f716e7","parentUid":"47316889e8fa06a3c4dea1c03cee6b4f","status":"failed","time":{"start":1726623391700,"stop":1726623402081,"duration":10381},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":true,"parameters":[],"tags":["asyncio","abort_on_fail","group(1)"]}],"uid":"47316889e8fa06a3c4dea1c03cee6b4f"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}